src/bulk-load.ts

```diff
@@ -508,6 +519,22 @@ class BulkLoad extends EventEmitter {
       addOptions.push('TABLOCK');
     }
 
+    if (this.bulkOptions.order) {
+      const orderColumns = [];
+
+      for (const column in this.bulkOptions.order) {
+        if (typeof column !== 'string') {
+          throw new TypeError('The "options.order.columnName" property must be of type string.');
+        }
+        // Ensure column exists as a specified column, otherwise we ignore.
+        orderColumns.push(`${column} ${this.bulkOptions.order[column]}`);
+      }
+
+      if (orderColumns.length) {
+        addOptions.push(`ORDER (${orderColumns.join(', ')})`);
+      }
+    }
+
```

Comment:
Can you move the validation logic into the constructor, and just leave the logic that builds the order fragment in here?

Throwing an error in this method is a bit unergonomic, because the error is thrown far away from where the actual problem lies. 😅

Reply:
Also, if you're iterating over the object key/values, I think it's better to use something like:

```js
for (const [column, order] of Object.entries(this.bulkOptions.order)) {
  ...
}
```

This ensures that prototype properties are skipped correctly.

Reply:
Makes sense!